bluebrain/repo-bluebrain/packages/py-atlas-direction-vectors/package.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAtlasDirectionVectors(PythonPackage):
+    """Commands to compute direction vectors in volumetric brain regions."""
+    homepage = "https://github.com/BlueBrain/atlas-direction-vectors"
+    git      = "git@github.com/BlueBrain/atlas-direction-vectors.git"
+    pypi     = "atlas-direction-vectors/atlas-direction-vectors-0.1.2.tar.gz"
+
+    version('0.1.2', tag='d87ff0caa7e03f62fa03fcd3230c398c3f74f5b7f550ac204098befbba6b8f45')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-atlas-commons@0.1.4:', type=('build', 'run'))
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('py-click@7.0:', type=('build', 'run'))
+    depends_on('py-numba', type=('build', 'run'))
+    depends_on('py-numpy@1.15.0:', type=('build', 'run'))
+    depends_on('py-numpy-quaternion', type=('build', 'run'))
+    depends_on('py-scipy@1.4.1:', type=('build', 'run'))
+    depends_on('py-voxcell@3.0.0:', type=('build', 'run'))
+    depends_on('regiodesics@0.1.0:', type='run')
```

Comment:
I don't see `regiodesics` in the installation instructions, is it needed but missing from the instructions?

Reply:
can we (in the future) then make that a variant of some sorts? Or add a comment here regarding optionality.

Reply:
this is the only variant that we need at bbp, since we have access to regiodesics; I'll make a comment